src/openms_gui/source/VISUAL/Spectrum3DCanvas.cpp

```diff
@@ -199,10 +199,10 @@ namespace OpenMS
       {
         openglwidget()->updateIntensityScale();
       }
-      openglwidget()->initializeGL();
+      // openglwidget()->initializeGL();
```

Comment:
yes, I think initializing opengl twice is what caused issues with my drivers here